exercises/protein-translation/canonical-data.json

```diff
@@ -207,6 +241,74 @@
         "strand": "UGGUGUUAUUAAUGGUUU"
       },
       "expected": ["Tryptophan","Cysteine","Tyrosine"]
+    },
+    {
+      "uuid": "1e75ea2a-f907-4994-ae5c-118632a1cb0f",
+      "description": "Non-existing codon can't translate",
+      "comments": [
+        "It's up to a track how to behave when a non-existing codon is    ",
+        "passed; to simplify the exercise, this test may be excluded in   ",
+        "order to remove the requirement for input validation.            "
+      ],
+      "scenarios": ["input-validation"],
+      "property": "proteins",
+      "input": {
+        "strand": "AAA"
+      },
+      "expected": {
+        "error": "Invalid codon"
+      }
+    },
+    {
+      "uuid": "9eac93f3-627a-4c90-8653-6d0a0595bc6f",
+      "description": "Unknown amino acids, not part of a codon, can't translate",
+      "comments": [
+        "It's up to a track how to behave when a character is encountered ",
+        "that doesn't represent one of the amino acids part of one of the ",
+        "codons. To simplify the exercise, this test may be excluded in   ",
+        "order to remove the requirement for input validation.            "
+      ],
+      "scenarios": ["input-validation"],
+      "property": "proteins",
+      "input": {
+        "strand": "XYZ"
+      },
+      "expected": {
+        "error": "Invalid codon"
+      }
+    },
+    {
+      "uuid": "9d73899f-e68e-4291-b1e2-7bf87c00f024",
+      "description": "Incomplete RNA sequence can't translate",
```

Comment:
```suggestion
      "description": "Incomplete RNA sequence results in an error",
```

Reply:
I think "can't translate" gives the reason why it errors rather than a more generic "results in an error" which we might already know if there is an error raised.

Reply:
I agree with kotp here.

Reply:
That's why I didn't say "results in an error". I expected maintainers to use the appends to get the behavior described.

Reply:
Seems like the consensus is against me here 😄.
I still think I'm correct, but I don't think it's a big enough issue to hold up the PR, so I remove my request for change 👍

Reply:
To be fair, you are not wrong, and if it were for a single track, then I could recommend doing it that way.

The comments that follow indicates that it may not raise an error, and so it would be in conflict with the comment.  It is up to the track to decide what to do with 'can't translate' and while raising an error is indeed one correct way to do this, it is not the only correct way.

Reply:
I agree. You're not wrong, but it's the problem of writing this for tracks that don't even have errors.